server/middlewares/auth.js

```diff
@@ -0,0 +1,13 @@
+const { verify } = require('jsonwebtoken');
```

Comment:
why you use destructing here ??  I think you can use it directly and please change the name, cause there is already method hold `verify` name
https://www.npmjs.com/package/jsonwebtoken

```suggestion
const  jwt  = require('jsonwebtoken');
```